.travis.yml

```diff
@@ -78,12 +77,9 @@ addons:
     - intltool
 before_script:
 - sudo mount --make-rshared /
-- bash -x ci/prepare-host $CPLATFORM $CVER
-- bash -x ci/prepare-host virtctl $KUBEVIRT_VER
-- bash -x ci/prepare-host oc 3.10.0
-- bash -x ci/start-cluster $CPLATFORM $CVER
-- bash -x ci/deploy-kubevirt $CPLATFORM $KUBEVIRT_VER
-- kubectl get pods --all-namespaces
-- kubectl describe nodes
-- kubectl describe -n kubevirt daemonset virt-handler
-- kubectl -n kubevirt logs -l kubevirt.io=virt-handler
```

Comment:
do we know why these were there in the first place?

Reply:
These logs were used for monitoring cluster and kubevirt health in early stage. We already have checks in traviscube + makefile tests, so now they are unnecessary.

Reply:
OK, fair enough